src/main/resources/lang/strings_en.properties

```diff
@@ -325,6 +325,8 @@ gui.map.loading=Loading
 gui.map.waiting=WAITING {0}
 gui.map.running=RUNNING {0}
 
+gui.tray_menu.quit=Quit DarkBot
```

Comment:
pretty weird place to place this, probs better to place it after gui.tray_button